osutil/export_test.go

```diff
@@ -194,12 +195,15 @@ func MockEtcFstab(text string) (restore func()) {
 
 // MockUname mocks syscall.Uname as used by MachineName and KernelVersion
 func MockUname(f func(*syscall.Utsname) error) (restore func()) {
+	/* The old code left for comparison
```

Comment:
It's fine to remove it in the PR, we will see it in the diff :)

Reply:
There was a politician that once unintentionally said a marvelous sentence "We wanted to make it better, but it went as always". :laughing:   

I think I did just that. I was trying to put more context for discussion, but eventually only confused everyone with redundancy. :flushed:   

I am sorry guys. Will learn it for future.